src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Order.xml

```diff
@@ -25,62 +25,70 @@
         <attribute name="validation_groups">sylius</attribute>
 
         <collectionOperations>
-            <collectionOperation name="get">
-                <attribute name="security">is_granted('ROLE_API_ACCESS')</attribute>
+            <collectionOperation name="admin_get">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">admin/orders</attribute>
             </collectionOperation>
-            <collectionOperation name="post">
-                <attribute name="security">is_granted('IS_AUTHENTICATED_ANONYMOUSLY')</attribute>
+
+            <collectionOperation name="shop_post">
+                <attribute name="method">POST</attribute>
+                <attribute name="path">/admin/orders</attribute>
+                <attribute name="messenger">input</attribute>
+                <attribute name="input">Sylius\Bundle\ApiBundle\Command\Cart\PickupCart</attribute>
                 <attribute name="openapi_context">
                     <attribute name="summary">Pickup a new cart</attribute>
                 </attribute>
-                <attribute name="messenger">input</attribute>
-                <attribute name="input">Sylius\Bundle\ApiBundle\Command\Cart\PickupCart</attribute>
             </collectionOperation>
         </collectionOperations>
 
         <itemOperations>
-            <itemOperation name="get" />
-            <itemOperation name="delete">
+            <itemOperation name="admin_get">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/admin/orders/{id}</attribute>
+            </itemOperation>
+
+            <itemOperation name="shop_get">
+                <attribute name="method">GET</attribute>
+                <attribute name="path">/shop/orders/{id}</attribute>
+            </itemOperation>
+
+            <itemOperation name="shop_delete">
+                <attribute name="method">DELETE</attribute>
+                <attribute name="path">/shop/orders/{id}</attribute>
```

Comment:
Is `DELETE` needed on the customer site?

Reply:
Yes, customer can delete cart (i mean `order` in state `cart`)

Reply:
Cart clear should work with this method. However, we should check if you can remove already placed order